src/components/ec-blog-item/ec-blog-item.vue

```diff
@@ -0,0 +1,51 @@
+<template>
+  <div
+    class="ec-blog-item tw-grid tw-border-b tw-border-solid tw-border-gray-6"
```

Comment:
I think you should move these tw classes in your `ec-blog-item` class. Unless I'm wrong I remember we agreed to keep all the tw stuff in a bem class. Correct me if Im wrong

Reply:
Same as above, actually you're replicating the markup, you should do it here and use the `<ec-blog-item>` tag above